src/calculateRentalCost.js

```diff
@@ -5,6 +5,25 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const shortTime = 2;
+  const avarageTime = 3;
+  const longTime = 7;
+  const costOfDay = 40;
+  const avarageTimeDiscount = 20;
+  const longTimeDiscount = 50;
+  const summ = days * costOfDay;
+
+  if (days >= longTime) {
+    return summ - longTimeDiscount;
+  }
```

Comment:
You have a bug here. According to the task, if the number of days is 7 or more, you should apply a $50 discount. However, your code applies the discount for 7 and more days. You should use the '>' operator instead of '>='.